clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -553,7 +586,8 @@ public ConsumerRebalanceListener rebalanceListener() {
         private boolean paused;  // whether this partition has been paused by the user
         private OffsetResetStrategy resetStrategy;  // the strategy to use if the offset needs resetting
         private Long nextRetryTimeMs;
-
+        private Integer preferredReadReplica;
+        private Long preferredReadReplicaLeaseMs;
```

Comment:
Would `preferredReadReplicaExpireTimeMs` be any clearer?

Reply:
Yea. "leaseMs" sounds like the lease duration rather than the expiration time (which it is)